controllers/pokemon_controller.go

```diff
@@ -0,0 +1,19 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/rromodev/academy-go-q32021/queries"
+
+	"github.com/gin-gonic/gin"
+)
+
+func GetPokemon(c *gin.Context) {
+	idp := c.Param("id")
+	id, _ := strconv.Atoi(idp)
+	fmt.Println(id)
+	pokemon := queries.GetPokemonById(id)
+	c.IndentedJSON(http.StatusOK, pokemon)
```

Comment:
If there was no pokemon found, then it would be better to return a NotFound status code, with a message like `Pokemon not found`